contrib/pylibpod/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright 2018 Red Hat, Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
```

Comment:
This paragraph should not be indented.  And you might want to just copy the root `LICENSE` file here anyway.